docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.1'
+
+networks:
+  db-data-net:
+    driver: bridge
+
+
+services:
+
+  db:
+    image: fredisson11/mysql-local:1.0.0
```

Comment:
The image name 'fredisson11/mysql-local:1.0.0' seems unusual. Please verify that this image is available and correctly spelled. Typically, a MySQL image from Docker Hub would look like 'mysql:5.7' or similar.